plugins/GSdx/res/tfx.fx

```diff
@@ -607,11 +610,15 @@ VS_OUTPUT vs_main(VS_INPUT input)
 {
 	if(VS_BPPZ == 1) // 24
 	{
-		input.z = input.z & 0xffffff; 
+		input.z = input.z & uint(0xffffff); 
 	}
 	else if(VS_BPPZ == 2) // 16
 	{
-		input.z = input.z & 0xffff;
+		input.z = input.z & uint(0xffff);
+	}
+	else
+	{
+		input.z;
```

Comment:
This does nothing.

Reply:
What about gl code, isn't it the same as well ?

Reply:
No. The commit you referenced fixed an uninitialised variable issue.

Reply:
Ah ok, my main reason for opening was this change.